### ISSUE TITLE
ELECTRA fine-tuning

### ISSUE THREAD

**Issue by Jaehun-Kim22** [Jaehun-Kim22/SCIC_Project#16]
# project interpretation (https://github.com/monologg/KoELECTRA)

1. pretrain/run_finetuning.py 사용해서 transformer 뒤에 추가적으로 레이어 붙이는 방식 적용
   1. tf.estimator 공부 필요 (initial thoughts: loss, optimizer, model을 모두 가지고 있고 학습에 필요한 과정을 쉽게 도와주는 API)
